### PR TITLE
Add docs on how to run carthage from Xcode.

### PR DESCRIPTION
**Pull request by guidomb**
I think this could be pretty useful for future contributors to know how to setup Carthage for debugging and being able to test it on "real" projects.

Thanks @mdiep for the tips.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,6 +35,29 @@ sudo make uninstall
 brew install carthage
 ```
 
+## Run Carthage from XCode
+
+I may be useful to run Carthage on different `Cartfile`s or in "real" project for debugging purpouse or to learn how Carthage works. Carthage is a regular macOS application so you can run it inside XCode and set breakpoints! To do so you may want to change the arguments passed to Carthage's command line tool and probably also change the working directory.
```
Comment (mdiep) [inline CONTRIBUTING.md]: First word here should be _It_.

**Comment by guidomb** [comment]
@mdiep Fixed

**Comment by guidomb** [comment]
@ikesyo Fixed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -35,6 +35,33 @@ sudo make uninstall
 brew install carthage
 ```
 
+## Run Carthage from Xcode
+
+<<<<<<< HEAD
+It may be useful to run Carthage on different `Cartfile`s or in "real" project for debugging purpouse or to learn how Carthage works. Carthage is a regular macOS application so you can run it inside Xcode and set breakpoints! To do so you may want to change the arguments passed to Carthage's command line tool and probably also change the working directory.
+=======
+I may be useful to run Carthage on different `Cartfile`s or in "real" project for debugging purpose or to learn how Carthage works. Carthage is a regular macOS application so you can run it inside Xcode and set breakpoints! To do so you may want to change the arguments passed to Carthage's command line tool and probably also change the working directory.
+>>>>>>> Fix typo.
```
Comment (ikesyo) [inline CONTRIBUTING.md]: Could you please fix the conflict?

**Comment by mdiep** [comment]
Thanks @guidomb! This is great!

**Pull request merged**